.github/workflows/release.yaml

```diff
@@ -87,7 +94,12 @@ jobs:
         run: |
           git config --global --add safe.directory "$(pwd)"
           . $HOME/.cargo/env
-          cargo build --release --target ${{ matrix.configs.target }}
+          SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct)
+          RUSTFLAGS="-C target-feature=+crt-static -C link-arg=-Wl,--build-id=none -Clink-arg=-static-libgcc -C metadata='' --remap-path-prefix $(pwd)=." \
```

Comment:
I feel it's important to explain exactly what each of these flags does. I like the way it's done in mev-boost, which is adding flags one by one, with a comment for each: https://github.com/flashbots/mev-boost/blob/develop/Makefile#L4-L15